mulog-kinesis/src/com/brunobonacci/mulog/publishers/kinesis.clj

```diff
@@ -100,7 +102,7 @@
    :format    :json
    ;; function to transform records
    :transform               identity
-   :kinesis-client-config   {:api  :kinesis}
+   :kinesis-client-config   {:api  :kinesis :region AWS-DEFAULT-REGION}
```

Comment:
I don't think we should set a default region here. The Default region is already set in the AWS SDK